src/python/build_flags.py

```diff
@@ -95,7 +95,7 @@ def regulatory_domain_to_env():
     regions = [("AU_915", "AU915"), ("EU_868", "EU868"), ("IN_866", "IN866"), ("AU_433", "AU433"), ("EU_433", "EU433"), ("FCC_915","FCC915"), ("ISM_2400", "ISM2G4"), ("EU_CE_2400", "CE2G4")]
     retVal = "UNK"
     if ("_2400" in target_name or \
-        '-DRADIO_2400=1' in build_flags) and \
+        '-DRADIO_SX1280=1' in build_flags) and \
```

Comment:
Doesn't this fail to work now, since RADIO_SX1280 is defined in the source target file, and not in build_flags any more?

Reply:
No, because the target header files are included in the .ini files